### PR TITLE
feat: update all dependencies that require node 10.x

### PR DESCRIPTION
**Pull request by clarkdo**
Checklist: https://github.com/nuxt/nuxt.js/issues/8327

### CODE REVIEW

File: packages/webpack/src/builder.js [hunk 1]
```diff
@@ -168,15 +168,19 @@ export class WebpackBundler {
     const buildOptions = this.buildContext.options.build
     const { client, ...hotMiddlewareOptions } = buildOptions.hotMiddleware || {}
 
+    compiler.options.watchOptions = this.buildContext.options.watchers.webpack
+    compiler.hooks.infrastructureLog.tap('webpack-dev-middlewre-log', (name) => {
```
Comment (kevcodez) [inline packages/webpack/src/builder.js:172]: Misspelled middleware, fyi

**Review by w3bdesign — DISMISSED** [review]

**Review by pi0 — DISMISSED** [review]

**Comment by clarkdo** [comment]
@pi0 This pr will result in some config change because of the packages breaking change like `css-loader`, I know you mentioned release note, it would be a great idea that list all the points.

**Comment by pi0** [comment]
@clarkdo Sure thing. Mainly merged to try on edge for a while and prepare list on same time. We may try to normalize some options probably?

**Comment by clarkdo** [comment]
I’m not sure if we should as some packages has renamed or reorganized configs in a better way, I think aligning to their official config may be more consistent

**Comment by pi0** [comment]
Just for backwards compatibility with a warning :)

**Comment by clarkdo** [comment]
Good idea, I can try to add some.

**Pull request merged**